ci/infra/testrunner/checks/checks.py

```diff
@@ -73,6 +73,13 @@ def __init__(self, conf, platform):
 
 
     def check_node(self, role, node, checks=None, stage=None, timeout=180, backoff=20):
+
+        #Prevent defaults to be accidentally overridden by callers with None
```

Comment:
I am curious to track where it happens. But the change seems sane to me.

Reply:
In case of the bootstrap fixture it calls skuba node_bootstrap without a timeout which then defaults to None. In turn, this function calls check_node passing its timeout (None).

The issue is hard to solve because either we provide a sensible default for every function that calls check_node or those functions should not pass the timeout unless they are provided a valid value. I plaayed with both options but neither one is safe. There's a better option I would probably implement which is a wrapper that removed parameters with value None if they have a default value. Stay tuned.